resources/views/tasks/edit.blade.php

```diff
@@ -533,6 +534,38 @@ class="multiselect__tag-icon"></i>
           },
         },
         methods: {
+          reassign() {
+            // The user shouldn’t be able to reassign the the current user is the only one in the pool of applicable users.
+            if (window.ProcessMaker.user.id == this.task.definition.assignedUsers) {
```

Comment:
My config includes:
1. The same user.
2. The `Requesters` group.

It seems that this is causing me to be unable to reassign anyone from the requesters group. 
I think I should be able to reassign someone from the requesters group.

Reply:
Thank you, this has been resolved!